torntools/views/settings/settings.js

```diff
@@ -70,6 +70,7 @@ requireDatabase(false)
 				},
 			});
 			message("Target list reset.", true);
+			location.reload();
```

Comment:
`message("Target list reset.", true, {reload: true });` does the same, but with a delay so they can read the message first.

Reply:
Done.